src/Yoti.Auth/AttributeConverter.cs

```diff
@@ -47,6 +50,56 @@ public static YotiAttribute<object> ConvertAttribute(AttrpubapiV1.Attribute attr
                         ParseAnchors(attribute));
         }
 
+        public static BaseAttribute ConvertToBaseAttribute(AttrpubapiV1.Attribute attribute)
+        {
+            YotiAttributeValue value;
+
+            switch (attribute.ContentType)
+            {
+                case ContentType.String:
+                    value = new YotiAttributeValue(TypeEnum.Text, attribute.Value.ToByteArray());
+                    return new YotiAttribute<string>(
+                      attribute.Name,
+                      value,
+                      ParseAnchors(attribute));
+
+                case ContentType.Date:
+                    value = new YotiAttributeValue(TypeEnum.Date, attribute.Value.ToByteArray());
+                    return new YotiAttribute<DateTime>(
+                        attribute.Name,
+                        value,
+                        ParseAnchors(attribute));
+
+                case ContentType.Jpeg:
+                    value = new YotiAttributeValue(TypeEnum.Jpeg, attribute.Value.ToByteArray());
+                    return new YotiAttribute<Image>(
+                        attribute.Name,
+                        value,
+                        ParseAnchors(attribute));
+
+                case ContentType.Png:
+                    value = new YotiAttributeValue(TypeEnum.Png, attribute.Value.ToByteArray());
+                    return new YotiAttribute<Image>(
+                        attribute.Name,
+                        value,
+                        ParseAnchors(attribute));
+
+                case ContentType.Json:
+                    value = new YotiAttributeValue(TypeEnum.Json, attribute.Value.ToByteArray());
+                    return new YotiAttribute<IEnumerable<Dictionary<string, JToken>>>(
+                        attribute.Name,
+                        value,
+                        ParseAnchors(attribute));
+
+                case ContentType.Undefined:
```

Comment:
Maybe you could combine these 2 into default?

Reply:
I guess it might not be as clear that the comment pertains to undefined attributes if we were to couple it with the default case.